### PR TITLE
sometimes my knees are as stiff as a metal pole

### PR DESCRIPTION
**Pull request by TiberianErin**
now CDDA characters can experience my occasional suffering

#### Summary
Content "Stiff Knees allow humans to suffer when they have to move over an obstacle"

#### Purpose of change
This PR adds a new trait which is a modification of Bad Knees, which acts as an even worse bad knees trait. It gives more points but also makes your legs even more problematic when you face uneven terrain and/or stairs. I dont think stairs actually act as obstacles, maybe that would be a future PR?

#### Describe the solution
I copied the normal bad knees trait definition, made a new trait underneath it, changed the ID from BADKNEES to REALLYBADKNEES (to prevent wacky crap and the game thinking that the two traits are the same), changed the name, added a funny description and ramped the normal 1.25 modifier up to 3.5, which is big enough to make walking through a bush take 10 seconds. That is a long time, just to get through a bush.

#### Describe alternatives you've considered
An alternative to this trait is probably applying a group of different mutations that would raise the obstacle movement multiplier to 3.5 together. But that is too hard compared to just, making your body worse.
Another alternative is just hurting your legs, probably. Small fires accomplish this, but they kind of go too far. (breaking your legs entirely, turning you into a [paraplegic](https://youtu.be/SKByTOTpVaM))

#### Testing
I downloaded the mutations.json file, shoved it into my copy of the newest experimental build, played it, got the stiff knees trait, and spent 10 seconds walking through a bush. I can confirm that this does function as expected. I am satisfied with my pole legs.

#### Additional context
image

### CODE REVIEW

File: data/json/mutations/mutations.json [hunk 1]
```diff
@@ -1278,18 +1278,29 @@
...
+    "cancels": [ "STRONGKNEES","REALLYBADKNEES" ],
     "valid": false,
     "movecost_obstacle_modifier": 1.25
   },
+  {
+    "type": "mutation",
+    "id": "REALLYBADKNEES",
+    "name": { "str": "Stiff Knees" },
+    "points": -3,
+    "description": "Your knees never bend. Period. Stiff as a metal pole. Pray that your journey doesnt involve stairs or slightly uneven terrain.",
```
Comment (GuardianDll) [inline data/json/mutations/mutations.json:1290]:
```suggestion
    "description": "Your knees never bend.  Period.  Stiff as a metal pole.  Pray that your journey doesn't involve stairs or slightly uneven terrain.",
```

File: data/json/mutations/mutations.json [hunk 2]
```diff
@@ -1278,18 +1278,29 @@
     "points": -1,
     "description": "Whether due to injury or age, your knees aren't particularly strong or flexible.  Moving over rough terrain will slow you down more than normal.",
     "starting_trait": true,
-    "cancels": [ "STRONGKNEES" ],
+    "cancels": [ "STRONGKNEES","REALLYBADKNEES" ],
```
Comment (GuardianDll) [inline data/json/mutations/mutations.json:1281]:
```suggestion
    "cancels": [ "STRONGKNEES", "REALLYBADKNEES" ],
```

File: data/json/mutations/mutations.json [hunk 3]
```diff
@@ -1278,18 +1278,29 @@
...
     "movecost_obstacle_modifier": 1.25
   },
+  {
+    "type": "mutation",
+    "id": "REALLYBADKNEES",
+    "name": { "str": "Stiff Knees" },
+    "points": -3,
+    "description": "Your knees never bend. Period. Stiff as a metal pole. Pray that your journey doesnt involve stairs or slightly uneven terrain.",
+    "starting_trait": true,
+    "cancels": [ "STRONGKNEES","BADKNEES" ],
```
Comment (GuardianDll) [inline data/json/mutations/mutations.json:1292]:
```suggestion
    "cancels": [ "STRONGKNEES", "BADKNEES" ],
```

File: data/json/mutations/mutations.json [hunk 4]
```diff
@@ -1278,18 +1278,29 @@
...
+  },
   {
     "type": "mutation",
     "id": "STRONGKNEES",
     "name": { "str": "Strong Knees" },
     "points": 1,
     "description": "Your knees are strong and capable of angles that you couldn't imagine previously.  Moving over rough terrain will slow you down less than normal.",
     "category": [ "BATRACHIAN" ],
-    "cancels": [ "BADKNEES" ],
+    "cancels": [ "BADKNEES","REALLYBADKNEES" ],
```
Comment (GuardianDll) [inline data/json/mutations/mutations.json:1303]:
```suggestion
    "cancels": [ "BADKNEES", "REALLYBADKNEES" ],
```

**Review by GuardianDll — APPROVED** [review]
👍

**Comment by TiberianErin** [comment]
thank you @GuardianDll for the help!

**Comment by TiberianErin** [comment]
@GuardianDll how long until maintainers see this?

**Comment by Zireael07** [comment]
@TiberianEuan Anybody's guess. There's few maintainers and many PRs

**Comment by TiberianErin** [comment]
> @TiberianEuan Anybody's guess. There's few maintainers and many PRs

damn that might take a while

**Pull request merged**